pysgrid/bld.bat

```diff
@@ -0,0 +1 @@
+"%PYTHON%" setup.py install --single-version-externally-managed  --record record.txt
```

Comment:
You have to remove the ` --single-version-externally-managed --record record.txt` here too.